docs/book/src/introduction/sway_quickstart.md

```diff
@@ -202,6 +202,34 @@ Note the contract ID — you will need it if you want to build out a frontend to
 
 If `forc-deploy` emits a `no such subcommand` error, try to install it with [fuelup](../introduction/installation.md) or [manually](https://github.com/FuelLabs/sway/releases).
 
+## Deploying the contract with Salt
+
+You can now pass a --salt flag in your forc deploy command to redeploy a contract that has been deployed already. This is useful for multiple deployments of the same contract.
+
+In order to use the `--salt` flag, install the latest `forc` update by running the following command in your project directory:
+
+```bash
+cargo add forc
+```
+
+Alternately, you can add the following to your `cargo.toml` file:
+
+```toml
+forc = "0.35.2"
+```
+
+Once `forc` is updated, can now add a `--salt` to your contractID while deploying the contract as follows:
```

Comment:
On a similar comment with above, updating `forc` alone would not suffice as what install `forc-deploy` is `forc-client`

Reply:
That's a very fair point. Taking both first and second comment into account, is it safe to skip any installation steps in this section all together and assuming that the reader has already installed the latest dependencies at the beginning of the quickstart during the installation step?

Reply:
I think it would be perfectly safe to assume that as we have an entire section for installation as the previous section. If you want to be extra cautious, you could add a note that mentions for to be able to use `forc deploy` or `forc run`  installing `forc-client` is needed.

Reply:
Awesome!! thanks for the notes. Will add final changes now.